src/main/java/be/woutschoovaerts/mollie/handler/AbstractHandler.java

```diff
@@ -172,4 +166,15 @@ protected void validateResponse(HttpResponse<String> response) throws IOExceptio
                     }));
         }
     }
+
+    private Map<String, String> getHeaderMap() {
```

Comment:
Maybe rename this method to **configureAllHeaders()**  it will be a bit more readable that way.